src/lib/eslint/BaseEslintEngine.ts

```diff
@@ -252,6 +254,17 @@ export abstract class BaseEslintEngine implements RuleEngine {
 		}
 	}
 
+	getNormalizedSeverity(severity: number): Severity {
+		switch (severity) {
+			case 1: {
+				return Severity.MODERATE;
+			}
+			case 2: {
+				return Severity.HIGH;
+			}
+		}
```

Comment:
Case statements should have a `default:` block. It's possible that an engine will add a new severity or there could be a bug somewhere that results in an unexpected value getting passed to the method.

The options for the default could be 1. return Severity.HIGH and maybe log a warning, 2.throw an exception. @rmohan20 - thoughts on which would be best?

Code style: I think the `{` and `}` can be removed. It might make it easier to read

Reply:
@jbartolotta-sfdc Good catch. @gjung-sfdc has added a default clause now. Since the user doesn't have any action on this exception, it would only block their run. Also, the warning would go to their logs and we won't know about it anyway. Severity.Moderate seems like a middle ground that doesn't break the user experience. What do you think?

Reply:
@rmohan20 I was thinking Severity.HIGH so that the user is made aware of it and can let us know. A Severity.Moderate might be missed by the user.